sre-recipes/recipes/ratings_freshness_recipe/ratings_freshness_recipe.py

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -*- coding: utf-8 -*-
+"""
+This module contains the implementation of rating2 recipe
+"""
+
+import logging
+import subprocess
+from recipe import Recipe
+
+
+class RatingsFreshnessRecipe(Recipe):
+    """
+    This class implements 'rating2' which purposefully
+    stops recollect API calls to rating service.
+    """
+
+    name = "recipe2"
+
+    def get_name(self):
+        return self.name
+
+    def is_active(self):
+        return True
+
+    def break_service(self):
+        """
+        Pause Cloud scheduler job to stop calls to recollect API of
+        the rating service
+        """
+        logging.info("Pausing scheduled job of the rating service")
+        print("Breaking rating operations...")
```

Comment:
Do you want to tell them which service is being broken? I thought that was part of what they were fining out

Maybe we should copy `recipe0` and say "Deploying broken service" instead

Reply:
yes, i will change it to a neutral message